bin-src/lib/startStorybook.ts

```diff
@@ -3,19 +3,22 @@ import { spawn } from 'cross-spawn';
 import path from 'path';
 import { Context } from '../types';
 
-export async function checkResponse(ctx: Context, url: string) {
+export async function resolveIsolatorUrl(ctx: Context, storybookUrl: string) {
   try {
-    // Allow invalid certificates, because we're running against localhost
-    await ctx.http.fetch(url, {}, { proxy: { rejectUnauthorized: false } });
-    return true;
+    // Allow invalid certificates, because we might be running against localhost.
+    const options = { proxy: { rejectUnauthorized: false } };
+    const { url: resolvedUrl } = await ctx.http.fetch(storybookUrl, {}, options);
+    const isolatorUrl = resolvedUrl.replace(/index\.html$/, '').replace(/\/?$/, '/iframe.html');
```

Comment:
Should we just drop anything with a file extension?

Reply:
I had considered that at first, but decided against it because that doesn't align with directory index behavior, and we might end up stripping a part of the path which was intentional. A directory name can technically look like a filename.